packages/frontend/src/pages/project/components/sections/KnowledgeNuggetsSection/KnowledgeNuggetsSection.tsx

```diff
@@ -20,7 +20,7 @@ export function KnowledgeNuggetsSection({
   knowledgeNuggets,
 }: KnowledgeNuggetsProps) {
   return (
-    <ProjectDetailsSection title={title} id={id} sectionOrder={sectionOrder}>
+    <Section title={title} id={id} sectionOrder={sectionOrder}>
```

Comment:
I would keep it `ProjectDetailsSection`. `Section` may lead to inproper interpretation of what that component does.